x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Observable } from 'rxjs';
+import moment from 'moment';
+import { OpsMetrics } from 'kibana/server';
+import { KIBANA_STATS_TYPE_MONITORING } from '../../../common/constants';
+
+/*
+ * Initialize a collector for Kibana Ops Stats
+ */
+export function getOpsStatsCollector(usageCollection: any, metrics$: Observable<OpsMetrics>) {
+  let lastMetrics: OpsMetrics | null = null;
+  metrics$.subscribe(metrics => {
+    lastMetrics = metrics;
+  });
+
+  return usageCollection.makeStatsCollector({
+    type: KIBANA_STATS_TYPE_MONITORING,
+    isReady: () => !!lastMetrics,
+    fetch: () => ({
+      ...lastMetrics,
+      timestamp: moment.utc().toISOString(),
```

Comment:
Should we maybe set the `timestamp` at the time when `lastMetrics` was reassigned (on line 18) rather than when it was requested with the fetch method? Since, that could be some time after the latest metrics were actually set

Reply:
Yea I wasn't sure either way, but happy to change it